labs/05_pom/models/home.py

```diff
@@ -1,6 +1,31 @@
+from playwright.sync_api import expect
+
+
 class HomePage:
     def __init__(self, page):
         self.page = page
+        self.input_product = self.page.locator("input[placeholder='Product Name']")
```

Comment:
Playwright also has a keyword to find by placeholder which is easier to use than this xpath locator